types/plotly.js/index.d.ts

```diff
@@ -434,6 +434,26 @@ export interface ModeBarButton {
 export type Datum = string | number | Date | null;
 export type TypedArray = Int8Array | Uint8Array | Int16Array | Uint16Array | Int32Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array;
 
+export type ErrorOptions = {
+	type: 'constant' | 'percent',
+	value: number,
+	visible?: boolean,
```

Comment:
Is it possible to condense these 5 shared properties (visible, color, thickness, width, opacity) into a separate shared type, and have these two possible ErrorOption definitions refer to that instead? Mainly to just ensure they are kept in sync - otherwise, looks great!